maxim/logger/livekit/llm.py

```
@@ -0,0 +1,97 @@
+        except Exception as e:
+            scribe().warning(
+                f"[Internal][{class_name}] {name} failed; error={e!s}\n{traceback.format_exc()}"
+            )
+            raise
+        finally:
+            post_hook(self, result, name, args, kwargs)
+
+    return functools.wraps(orig)(wrapper)
+
+
+def instrument_llm(orig, name):
```

Ruff (0.12.2)

Missing return type annotation for public function `instrument_llm`

(ANN201)